docs_app/tools/decision-tree-generator/src/tree.yml

```diff
@@ -253,21 +250,6 @@
             - label: and start it as soon as the first subscriber arrives
               children:
                 - label: share
-            - label: and start it manually or imperatively
-              children:
-                - label: publish
```

Comment:
I think `publish` in this scenario could be replaced with `connectable`.

Reply:
Done.